### PR TITLE
Fix issues

### PR DESCRIPTION
**Pull request by suaas21**

### CODE REVIEW

File: docs/guides/latest/advance-use-case/clone-pvc.md [hunk 1]
```diff
@@ -693,10 +693,10 @@ spec:
...
 - `spec.target.volumeClaimTemplates:` a list of PVC templates that will be created by Stash to restore the respective backed up data.
 
-  - `metadata.name` is a template for the name of the restored PVC that will be created by Stash. You have to provide this name template to match with your desired StatefulSet's PVC. For example, if you want deploy a StatefulSet named `stash-demo` with `volumeClaimTemplate` name `my-volume`, your StatefulSet's PVC will be`my-volume-stash-demo-0`, `my-volume-stash-demo-1` and so on. In this case, you have to provide `volumeClaimTemplate` name in RestoreSession in the following format:
+  - `metadata.name` is a template for the name of the restored PVC that will be created by Stash. You have to provide this named template to match with your desired StatefulSet's PVC. For example, if you want deploy a StatefulSet named `stash-demo` with `volumeClaimTemplate` name `my-volume`, your StatefulSet's PVC will be`my-volume-stash-demo-0`, `my-volume-stash-demo-1` and so on. In this case, you have to provide `volumeClaimTemplate` name in RestoreSession in the following format:
```
Comment (hossainemruz) [inline docs/guides/latest/advance-use-case/clone-pvc.md]: name

File: docs/guides/latest/workloads/deployment.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ namespace/demo created
 
 This section will show you how to use Stash to backup volumes of a Deployment. Here, we are going to deploy a Deployment with a PVC and generate some sample data in it. Then, we are going to backup this sample data using Stash.
 
-### Deploy workload
+### Prepare workload
```
Comment (hossainemruz) [inline docs/guides/latest/workloads/deployment.md]: Workload

File: docs/guides/latest/workloads/deployment.md [hunk 3]
```diff
@@ -481,7 +480,7 @@ Here,
 
 - `spec.repository.name` specifies the `Repository` crd that holds the backend information where our backed up data has been stored.
 
-- `spec.target.ref` refers to the target workload where the recovered data will be stored.
+- `spec.target.ref` refers to the target workload where the recover data will be stored.
```
Comment (hossainemruz) [inline docs/guides/latest/workloads/deployment.md]: recovered

File: docs/guides/latest/workloads/deployment.md [hunk 4]
```diff
@@ -490,7 +489,7 @@ $ kubectl apply -f ./docs/examples/workloads/deployment/restoresession.yaml
 restoresession.stash.appscode.com/deployment-restore created
 ```
 
-Once, you have created the `RestoreSession` crd, Stash will inject `init-container` to `stash-recovered` Deployment. The Deployment will restart and the `init-container` will recovered on start-up.
+Once, you have created the `RestoreSession` crd, Stash will inject `init-container` to `stash-recovered` Deployment. Deployment will restart and the `init-container` will restore desired data on start-up.
```
Comment (hossainemruz) [inline docs/guides/latest/workloads/deployment.md]: into

Comment (hossainemruz) [reply docs/guides/latest/workloads/deployment.md]: the desired data

File: docs/guides/latest/workloads/statefulset.md [hunk 5]
```diff
@@ -511,11 +510,11 @@ $ kubectl apply -f ./docs/examples/workloads/statefulset/restoresession.yaml
 restoresession.stash.appscode.com/ss-restore created
 ```
 
-Once, you have created the `RestoreSession` crd, Stash will inject `init-container` to `stash-recovered` StatefulSet. The StatefulSet will restart and the `init-container` will recovered on start-up.
+Once, you have created the `RestoreSession` crd, Stash will inject `init-container` to `stash-recovered` StatefulSet. The StatefulSet will restart and the `init-container` will recover on start-up.
```
Comment (hossainemruz) [inline docs/guides/latest/workloads/statefulset.md]: recover -> will restore the desired data

Comment (hossainemruz) [inline docs/guides/latest/workloads/statefulset.md]: inject `init-container` into `stash-recovered`

File: docs/guides/latest/workloads/statefulset.md [hunk 6]
```diff
@@ -634,7 +634,7 @@ stash-demo-2
 
 ### Customize Restore Process
 
-Generally, Stash restores data in individual replicas from a backup of respective replica of the original StatefulSet. That means, backed up data of `pod-0` of original StatefulSet will be restored in `pod-0` of new StatefulSet and so on. However, you can customize this behavior through `spec.rules` section of RestoreSession object. This is particularly helpful when your restored StatefulSet has different number of replicas than the original StatefulSet. You can control which data will be restored in the recovered replicas.
+Generally, Stash restores data in individual replicas from a backup of the respective replica of the original StatefulSet. That means, backed up data of `pod-0` of original StatefulSet will be restored in `pod-0` of new StatefulSet and so on. However, you can customize this behavior through the `spec.rules` section of RestoreSession object. This is particularly helpful when your restored StatefulSet has a different number of replicas than the original StatefulSet. You can control which data will be restored in the recovered replicas.
```
Comment (hossainemruz) [inline docs/guides/latest/workloads/statefulset.md]: restored in the recovered replicas. -> restored in the extra replicas.

File: docs/guides/latest/workloads/statefulset.md [hunk 7]
```diff
@@ -752,11 +752,11 @@ $ kubectl apply -f ./docs/examples/guides/latest/workloads/statefulset/adv_resto
 restoresession.stash.appscode.com/ss-restore created
 ```
 
-Once, you have created the `RestoreSession` crd, Stash will inject `init-container` to `stash-recovered` StatefulSet. The StatefulSet will restart and the `init-container` will recovered on start-up.
+Once, you have created the `RestoreSession` crd, Stash will inject `init-container` to `stash-recovered` StatefulSet. The StatefulSet will restart and the `init-container` will restore desired data on start-up.
```
Comment (hossainemruz) [inline docs/guides/latest/workloads/statefulset.md]: restore the desired data

Comment (hossainemruz) [inline docs/guides/latest/workloads/statefulset.md]: into

File: docs/guides/latest/advance-use-case/clone-pvc.md [hunk 8]
```diff
@@ -730,7 +730,7 @@ So, we can see from the output of the above command that the restore process suc
 
 **Verify Restored PVC:**
 
-Once a restore process is complete, verify that new PVCs have been created successfully by the following command,
+Once a restore process is complete, verify that new PVCs has been created successfully by the following command,
```
Comment (tamalsaha) [inline docs/guides/latest/advance-use-case/clone-pvc.md]: `PVC[s]` is plural. So, have was correct.

File: docs/guides/latest/advance-use-case/instant-backup.md [hunk 9]
```diff
@@ -224,7 +224,7 @@ deployment-backup          */40 * * * *            6m41s
 
 ## Trigger Instant Backup
 
-Now, we are going to trigger an instant backup of the volumes of the Deployment that we have configured in the previous section. In order to do that, we have to create a `BackupSession` object pointing to the respective `BackupConfiguration` object.
+Now, we are going to trigger an instant backup of the volumes of the Deployment that we have configured in the previous section. To do that, we have to create a `BackupSession` object pointing to the respective `BackupConfiguration` object.
```
Comment (tamalsaha) [inline docs/guides/latest/advance-use-case/instant-backup.md]: `In order to` and `To` are both correct and same meaning.

**Pull request merged**